src/sum.js

```diff
@@ -0,0 +1,3 @@
+export default function sum(a, b) {
+  return a + b;
+}
```

Comment:
I do not know what this sum function is being used for. You can explain please, If it is not required then we can do away with it at the moment.

Reply:
These are just dummy tests for Jest. So we'll remove them later